### ISSUE TITLE
task offset

### ISSUE THREAD

**Issue by vaniron** [pterodactyl/panel#3577]
### Is there an existing feature request for this?

- [X] I have searched the existing issues before opening this feature request.

### Describe the feature you would like to see.

task in schedule executed only after backup is finished

### Describe the solution you'd like.

make offset applied after previous task is finished, not from time it started

### Additional context to this request.

when making backup of whole server which got pregenerated maps, backuping takes quite some time and no idea how long can it take.. now task offset works with time when previous task was started, solution would be to make it count time after previous task (backup) has been finished so i can have sequence: stop the server => backup => => start the server

**Comment by trenutoo** [comment]
Duplicate of #2872
